### ISSUE TITLE
Error: Invalid JSON provided, when providing more than one key/value pair to pinList metadata filter

### ISSUE THREAD

**Issue by ltfschoen** [PinataCloud/Pinata-SDK#31]
If I define a metadata filter for `pinList` https://github.com/PinataCloud/Pinata-SDK#pinlist where there is only a single key/value pair, then it the filter works, however, if I add a second key/value pair to the metadata filter as used here https://github.com/ltfschoen/ethquad/commit/a0e3cebe70eb9949a0c74a96ae226ff59a260d86#diff-285c0d8921689baaf0445b8e23d2263eR15, then I get error:
```
[0] (node:10952) UnhandledPromiseRejectionWarning: Error: Invalid JSON provided
[0]     at /Users/ls/code/src/ltfschoen/ethquad/node_modules/@pinata/sdk/lib/pinata-sdk.js:21486:16
[0]     at processTicksAndRejections (internal/process/task_queues.js:93:5)
[0] (Use `node --trace-warnings ...` to show where the warning was created)
[0] (node:10952) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
[0] (node:10952) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by ltfschoen** [comment]
I get the error even if I use the example values of `metadata` shown here https://github.com/PinataCloud/Pinata-SDK#pinlist

**Comment by jehunter5811** [comment]
Hey @ltfschoen we're looking into this now. Thanks for the heads up. If you have an immediate need, you can fall back to direct API queries. But regardless, we'll keep you posted here.

**Comment by obo20** [comment]
@ltfschoen This should be fixed now with the newest release of 1.1.11
